db/models.py

```diff
@@ -1 +1,40 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
```

Comment:
As task requires:
Each player should choose a race to play, such as Elf, Dwarf, Human, or Ork.

So, you need to add argument choices to name field.

Reply:
also having RACES_NAME inside of the Race class would be better